README.md

```diff
@@ -117,16 +123,16 @@ There are two ways to send data to your analytics services through this library:
 
 #### Cloud-based Connection Modes
 
-When an destination’s SDK is not packaged, but it is enabled via your dashboard, the request goes through the Segment REST API, and is routed to the service’s server-side API as [described here](https://segment.com/docs/integrations/#connection-modes).
+When an destination’s SDK is not packaged, but it is enabled using the Segment web UI, the request goes through the Segment REST API, and is routed to the service’s server-side API as [described here](https://segment.com/docs/connections/destinations/#connection-modes).
 
-#### Packaging Device-based destination SDKs
+#### Packaging Device-mode Destination SDKs
```

Comment:
If we're going to make this change, we should make it to all instances of this term that refer to this in the context of how it sends the data. :)